R/readNewClimate.R

```diff
@@ -0,0 +1,111 @@
+#' Reads NPI policy database with technology capacity target from the Policy data base (v4 August 2024) 
+#' by PBL that translate the high impact policies of https://climatepolicydatabase.org/.
+#' @description Reads excel sheet with NPi (National Policies Implemented)
+#' data on different policy targets (capacity) with different variations: unconditional for minimal and conditionnal for maximum targets. 
+#' NPI targets only include targets that are based on implemented policy instruments.
+#' @details Country name is ISO coded. Capacity/Additional Capacity targets are in GW. 
+#' @return  magpie object
+#' @author  Rahel Mandaroux, Léa Hayez, Falk Benke
+#' @param subtype Capacity_2024_cond (or uncond) for capacity target,
+#' @importFrom readxl read_xlsx
+#' @importFrom dplyr select
+
+#############
+#for testing
+#setwd("C:/Users/rahelma/remind/sources/NewClimate")
+#NPIfile <- "NPi_2025-02-03.xlsx"
+#############
+
+readNewClimate <- function(subtype) {
+ # browser()
+ 
+  if (grepl("2025", subtype, fixed = TRUE)) { #keep structure to compare when new versions (NPi target updates) are generated
+    NPIfile <- "NPi_2025-02-03.xlsx"
+  } else {
+    NPIfile <- "NPi_2025-02-03.xlsx"
+    if (!grepl("2025", subtype, fixed = TRUE)) {
+      warning("\nNo data for year in ", subtype, " available. Choose default: ", NPIfile)
+    }
+  }
+  
+  if (grepl("Capacity", subtype, fixed = TRUE)) {
```

Comment:
the if clauses checking the subtype should be `if (..) {Do Capacitiy Part } else if(..) { Do Emissiosn Part } else { produce error}`